test/test_flash.py

```diff
@@ -125,6 +125,15 @@ def test_run_with_uppercase_HTM(self, mock_popen, mock_verifier, mock_copy_file)
         self.assertEqual(ret, 0)
         self.assertEqual(2, mock_out.call_count)
 
+    @unittest.skipIf(platform.system() != 'Windows', 'require windows')
+    @mock.patch('os.system')
+    def test_copy_file_with_spaces(self, mock_system):
+        flasher=FlasherMbed()
+        srcfile = "%s/test_flash.py" % os.path.abspath(os.curdir)
```

Comment:
os.curdir seems is `mbed-flasher/`, should be `mbed-flasher/test`

Reply:
and `/` might be a problem

Reply:
@YifeiZuo01 is this now ok?

Reply:
yes.